provider/internal/builtin/service_client.go

```diff
@@ -324,15 +324,19 @@ func (b *builtinServiceClient) getLocation(ctx context.Context, path, content st
 		line := strings.Trim(part, " ")
 		line = strings.ReplaceAll(line, "\t", "")
 		line = regexp.QuoteMeta(line)
-		if line != "" {
-			lines = append(lines, line)
-		}
+		lines = append(lines, line)
+	}
+	// remove leading and trailing empty lines
+	if len(lines) > 0 && lines[0] == "" {
+		lines = lines[1:]
 	}
-	if len(lines) < 1 {
-		return location, fmt.Errorf("unable to get code location, no-op content")
+	if len(lines) > 0 && lines[len(lines)-1] == "" {
+		lines = lines[:len(lines)-1]
+	}
+	if len(lines) < 1 || strings.Join(lines, "") == "" {
```

Comment:
@shawn-hurley this check should take care of the empty case

Reply:
+1